homeassistant/components/sensor/envirophat.py

```diff
@@ -53,7 +53,13 @@
 
 def setup_platform(hass, config, add_devices, discovery_info=None):
     """Set up the Sense HAT sensor platform."""
-    data = EnvirophatData(config.get(CONF_USE_LEDS))
+    try:
```

Comment:
@pvizeli this is actually the correct approach. The package is raising an error when it is being imported if it can't find the hardware. (it's not a good design of the package that import has side effects but that is out of the question here)